rest/client/timeout_sender.go

```diff
@@ -0,0 +1,51 @@
+package client
+
+import (
+	"github.com/mongodb/grip/message"
+	"github.com/mongodb/grip/send"
+)
+
+type timeoutLogSender struct {
+	sender send.Sender
```

Comment:
if you delete the field name from this line, you can capitalize the third non whitespace character on line 14 and then delete lines 17-44.